pkg/codegen/go/gen.go

```diff
@@ -1379,6 +1379,11 @@ func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error
 	return files, nil
 }
 
+// goPackage returns the suggested package name for the given string.
+func goPackage(name string) string {
+	return strings.Split(name, "-")[0]
```

Comment:
should we be defensive here and check to see if there are any items after the split?

Reply:
do you mean when `name` is `""`?

Reply:
Added a test case for `""`. I think `Split` would always return one item if a separator is specified.

Reply:
yeah